tools/sifgen/node/node_test.go

```diff
@@ -0,0 +1,348 @@
+package node
```

Comment:
Hey, why did you comment this out?  Isn't testing node.go relevant and useful?

Reply:
Hi @khdegraaf  , The node module and the sifgen tools were updated ,which makes this test fail . I checked with @intl-man , he has a separate task to rewrite /fix these scripts. 
https://app.asana.com/0/1196711589172801/1198934120261357

I commented this out to get things to compile, the new tests/ Fixes  can go in as a separate pr ,as its not directly related to clp. Would love to talk more though in case you feel we need to add it to this PR

Reply:
OK for now.